serializable_request.py

```diff
@@ -1,5 +1,48 @@
 import requests
 import json
+from urllib.parse import urlsplit, urlunsplit
+
+
+class Response():
+    def __init__(self, status, data, headers):
+        self.status = status
+        self.data = data
+        self.headers = headers
+
+    @classmethod
+    def from_file(cls, fname):
+        with open(fname, 'r') as f:
+            return cls.from_json(f.read())
+
+    @classmethod
+    def from_json(cls, json_data):
+        data = json.loads(json_data)
+        return cls(
+            int(data['status']),
+            data['data'].encode('utf-8'),
+            data['headers'],
+        )
+
+    @classmethod
+    def from_requests_response(cls, res: requests.Response):
+        return cls(res.status_code, res.content, dict(res.headers))
+
+    def to_file(self, fname):
+        with open(fname, 'w') as f:
+            f.write(self.as_json())
+
+    def as_json(self):
+        return json.dumps(self.as_dict())
+
+    def as_dict(self):
```

Comment:
```suggestion
    def to_dict(self):
```
Why is it `to_file` but `as_dict`? Let's be consistent.

Reply:
(the same as previous comment - there is some consistency :) )